src/current/_includes/releases/v23.2/v23.2.0-alpha.1.md

```diff
@@ -166,51 +177,78 @@ Release Date: September 23, 2023
 - Exposed a new metric `storage.compactions.duration`, computed by the storage engine, that provides the cumulative time the storage engine has spent in compactions. This duration may exceed time elapsed, because of concurrent compactions, and may be useful in monitoring compaction concurrency. [#103670][#103670]
 - The default value for the `--max-sql-memory` parameter of the [`cockroach demo` command]({% link v23.2/cockroach-demo.md %}) has been increased from 128 MiB to 256 MiB. [#103642][#103642]
 - Two new store metrics, `range.snapshots.cross-region.sent-bytes` and `range.snapshots.cross-region.rcvd-bytes`, were added to track the aggregate of snapshot bytes sent from and received at a store across different regions. Note that these metrics require the nodes' localities to include a “region” tier key. If a node lacks this key but is involved in cross-region batch activities, an error message will be logged. [#104111][#104111]
-- Two new store metrics, `range.snapshots.cross-zone.sent-bytes` and `range.snapshots.cross-zone.rcvd-bytes`, were added to track the aggregate of snapshot bytes sent from and received at a store across different zones.  For accurate metrics, follow these assumptions:
+- Added new store metrics to track the aggregate of snapshot bytes sent from and received at a store across different zones.
+
+  - `range.snapshots.cross-zone.sent-bytes`
+  - `range.snapshots.cross-zone.rcvd-bytes`
+
+   For accurate metrics, follow these recommendations:
```

Comment:
This isn't quite rendering properly. I think the "For accurate..." sentence needs to bump in a tab one more. and the same for the points following

Reply:
It is affecting the release notes that follow this one too.